Packs/Jira/Integrations/JiraV3/JiraV3.yml

```diff
@@ -1539,6 +1539,158 @@ script:
+    - contextPath: Jira.IssueType.entityId
+      description: Unique ID for next-gen projects.
+      type: String
+    - contextPath: Jira.IssueType.expand
+      description: Expand options that include additional issue type metadata details in the response.
+      type: String
+    - contextPath: Jira.IssueType.iconUrl
+      description: The URL of the issue type's avatar.
+      type: String
+    - contextPath: Jira.IssueType.id
+      description: The ID of the issue type.
+      type: String
+    - contextPath: Jira.IssueType.name
+      description: The name of the issue type.
+      type: String
+    - contextPath: Jira.IssueType.self
+      description: The URL of these issue type details.
+      type: String
+    - contextPath: Jira.IssueType.subtask
+      description: Whether this issue type is used to create subtasks.
+      type: Boolean
+    - contextPath: Jira.IssueType.scope
+      description: Details of the next-gen projects the issue type is available in.
+      type: Object
+    - contextPath: Jira.IssueType.scope.type
+      description: 'The type of scope. Valid values: PROJECT, TEMPLATE.'
+      type: String
+    - contextPath: Jira.IssueType.scope.project
+      description: The project the item has scope in.
+      type: Object
+    - contextPath: Jira.IssueType.scope.project.self
+      description: The URL of the project details.
+      type: String
+    - contextPath: Jira.IssueType.scope.project.id
+      description: The ID of the project.
+      type: String
+    - contextPath: Jira.IssueType.scope.project.key
+      description: The key of the project.
+      type: String
+    - contextPath: Jira.IssueType.scope.project.name
+      description: The name of the project.
+      type: String
+    - contextPath: Jira.IssueType.scope.project.projectTypeKey
+      description: 'The project type of the project. Valid values: software, service_desk, business.'
+      type: String
+    - contextPath: Jira.IssueType.scope.project.simplified
+      description: Whether or not the project is simplified.
+      type: Boolean
+    - contextPath: Jira.IssueType.scope.project.avatarUrls
+      description: The URLs of the project's avatars.
+      type: Object
+    - contextPath: Jira.IssueType.scope.project.projectCategory
+      description: The category the project belongs to.
+      type: Object
+    - contextPath: Jira.IssueType.scope.project.projectCategory.self
+      description: The URL of the project category.
+      type: String
+    - contextPath: Jira.IssueType.scope.project.projectCategory.id
+      description: The ID of the project category.
+      type: String
+    - contextPath: Jira.IssueType.scope.project.projectCategory.description
+      description: The name of the project category.
+      type: String
+    - contextPath: Jira.IssueType.scope.project.projectCategory.name
+      description: The description of the project category.
+      type: String
+  - name: jira-get-create-metadata-field
```

Comment:
It is a `get` action. You `get` the `creation metadata` fields. The naming is confusion, but it is according to the jira Rest API https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issues/#api-rest-api-3-issue-createmeta-projectidorkey-issuetypes-issuetypeid-get

Reply:
Ok.
Our naming convention for commands is `<vendor>-<object>-<operation>`, so let's call it `jira-create-metadata-issue-types-list` and `jira-create-metadata-field-get`.

Reply:
Why is one command with `-list` and the other with `-get`?

Reply:
Because issue-types sound like multiple objects, but metadata-field is one.
If in the metadata-field it could return more than one then let's do it list as well.

Reply:
Yes, `metadata-field` can return a list of fields. Will update the command now